host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent

```diff
@@ -0,0 +1,56 @@
+upstream glitchtip {
+   server localhost:8000;
```

Comment:
Is this port OK? Where should I run it on? Is there any overview on used ports or guidelines for new applications?